### ISSUE TITLE
Expose error types?

### ISSUE THREAD

**Issue by akerl** [gorilla/securecookie#51]
It seems like the ideal behavior for a cookie w/ an expired timestamp would be for me to consider it equivalent to http.ErrNoCookie, but right now I can't find a clean way to check for that, since the types of errors are all private.

Am I missing a better workflow or existing clean way to check for this?

**Comment by elithrar** [comment]
Agree. This is something I’d like to refactor—and simplify—under my v2
rewrite. Error handling in Go is never straightforward and I don’t think
the current approach in securecookie is ideal.

Moving to a v2 will still be slightly tricky, since there’s the conflict of
“change import path” vs. “break Go users without a version-aware go get” -
that’s something I need to assess in June.

**Comment by buchdag** [comment]
@elithrar any update about this ?

**Comment by akerl** [comment]
Any movement here?
